.eslintignore

```diff
@@ -0,0 +1 @@
+﻿_tests_
```

Comment:
Why would we not want to lint our test code?

Reply:
I have removed _tests_ from this file.